src/transformers/tools/base.py

```diff
@@ -344,6 +377,96 @@ def __init__(self, _gradio_tool):
 
         GradioToolWrapper.__call__ = gradio_tool.run
         return GradioToolWrapper(gradio_tool)
+    
+
+    @staticmethod
+    def from_langchain(langchain_tool):
```

Comment:
@Jofthomas what do you think of moving the method definition inside the class like this? (I copied the existing `from_gradio` method)

Reply:
seems fine to me :)